project/generate_dynamic_defs/dynamic_definition_builder.ts

```diff
@@ -0,0 +1,38 @@
+import { AssetSourceFile } from "../assets/asset_source_file"
+import { TsGdProject } from "../project"
+import buildActionNames from "./build_action_names"
+import buildAssetPathsType from "./build_asset_paths"
+import buildGroupTypes from "./build_group_types"
+import buildNodePathsTypeForScript from "./build_node_paths"
+import buildSceneImports from "./build_scene_imports"
+
+export default class DynamicDefinitionBuilder {
```

Comment:
Naming nit: maybe just `DefinitionBuilder`?

Reply:
Could do LibraryBuilder and DefinitionBuilder, yeah.